frontend/src/static/js/v_authorship.js

```diff
@@ -1,13 +1,19 @@
 window.toggleNext = function toggleNext(ele) {
-  // function for toggling unopened code
-  const target = ele.nextSibling.style;
+    // function for toggling unopened code
+    const target = ele.nextSibling.style;
+    const child = ele.firstChild.style;
```

Comment:
airbnb style guide uses 2-space indentation.

run `npm run lint -- --fix frontent/src/**/*js` from the project root, eslint will take care of most of the styling for you.